### PR TITLE
custom addIcon and removeIcon for ListFields for bootstrap-4

### PR DESCRIPTION
**Pull request by todda00**
Add ability to include custom add / remove icons for ListFields.

This can be implemented in either AutoField or by using ListField.

Example AutoField:
```
<AutoField
  name="athletics.otherSports"
  addIcon={(<i className="fa fa-add-circle" />)}
  removeIcon={(<i className="fa fa-close-circle" />)}
/>
```

Example ListField:
```
<ListField
  name="athletics.otherSports"
  addIcon={(<i className="fa fa-add-circle" />)}
  removeIcon={(<i className="fa fa-close-circle" />)}
>
  <ListItemField name="$">
    <NestField>
      <TextField name="sportName" className="col-md-6" />
      <TextField name="yearsParticipated" className="col-md-6" />
    </NestField>
  </ListItemField>
</ListField>
```

### CODE REVIEW

File: packages/uniforms-bootstrap4/src/components/fields/ListAddField.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const ListAdd = ({
     disabled,
     parent,
     value,
+    addIcon = (<i className="octicon octicon-plus" />),
```
Comment (radekmie) [inline packages/uniforms-bootstrap4/src/components/fields/ListAddField.js]: Please, use `defaultProps` instead.

File: packages/uniforms-bootstrap4/src/components/fields/ListDelField.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const ListDel = ({
     disabled,
     name,
     parent,
+    removeIcon = (<i className="octicon octicon-dash" />),
```
Comment (radekmie) [inline packages/uniforms-bootstrap4/src/components/fields/ListDelField.js]: Same as above.

File: packages/uniforms-bootstrap4/src/components/fields/ListField.js [hunk 3]
```diff
@@ -16,6 +16,8 @@ const List = ({
     label,
     name,
     value,
+    addIcon,
+    removeIcon,
```
Comment (radekmie) [inline packages/uniforms-bootstrap4/src/components/fields/ListField.js]: I know it's not in the ESLint, but please, keep them sorted.

**Comment by radekmie** [comment]
Great!
Maybe you could implement the same functionality in bootstrap3 package?

**Comment by todda00** [comment]
I don't use BS3, but it should be relatively simple, I will create a separate PR for BS3.

File: packages/uniforms-bootstrap4/src/components/fields/ListAddField.js [hunk 4]
```diff
@@ -24,4 +24,8 @@ const ListAdd = ({
     );
 };
 
+ListAdd.defaultProps = {
+    addIcon: (<i className="octicon octicon-plus" />),
```
Comment (radekmie) [inline packages/uniforms-bootstrap4/src/components/fields/ListAddField.js]: Sorry for being so picky _(just say no, if you want)_, but could you remove the parentheses and trailing comma?

Comment (todda00) [inline packages/uniforms-bootstrap4/src/components/fields/ListAddField.js]: sure thing, we each have our own style, I'll try to match yours. FYI my thought process behind the comma: I prefer commas after the final line so when you add a property at the end of an object, the commit is only 1 line change, not 2.

Comment (todda00) [inline packages/uniforms-bootstrap4/src/components/fields/ListAddField.js]: Do you want the defaultProps all one line (just 1 property), or multi ?

**Comment by radekmie** [comment]
I'll release `1.3.0` soon.

**Pull request merged**